packages/babel/package.json

```diff
@@ -16,7 +16,9 @@
   "main": "./dist/cjs/index.js",
   "module": "./dist/es/index.js",
   "exports": {
-    "types": "./types/index.d.ts",
+    "types": {
+      "import": "./dist/es/index.d.ts"
```

Comment:
such file doesn't exist: https://unpkg.com/browse/@rollup/plugin-babel@6.0.3/dist/es/

Reply:
I changed the original build configuration https://github.com/rollup/plugins/pull/1578/files#diff-ac31c1a22fbd2206b68c48201cc3989a1fe9e4b267c217c72c7fcb2efe5e8fb4R56

Reply:
Oh, cool! I actually looked for it but missed it while scrolling 😅